README.md

```diff
@@ -18,6 +17,8 @@ The ROS 2 Navigation System is the control system that enables a robot to autono
 # Documentation
 For detailed instructions on how to install and run the examples, please visit our [documentation site](https://ros-planning.github.io/navigation2/).
 
+Run `doxygen` in the root of this repository. It will generate a `/doc/*` directory containing the documentation. Entrypoint in a browser is index.html.
```

Comment:
This is confusing to me, doxygen is different than the github pages documentation, but this makes it seem like doxygen is how that documentation is generated. I think a sub-heading like 'API documentation' or something would be good to clarify the difference.

Reply:
@mkhansen-intel good point - done.